sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/ServiceBusTriggerAttribute.cs

```diff
@@ -89,6 +90,26 @@ public string SubscriptionName
         /// </summary>
         public bool IsSessionsEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets the flag whether trigger should automatically call complete after processing, or if the function code will manually call complete.
+        /// </summary>
+        public bool AutoCompleteMessages
+        {
+            get
+            {
+                return _autoCompleteMessages.HasValue ? _autoCompleteMessages.Value : true;
+            }
+            set
+            {
+                _autoCompleteMessages = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets a boolean to check if auto complete messages option was set on the trigger.
```

Comment:
Can we add a comment explaining why this is needed, since we can't use a nullable property in the attribute?